README.md

```diff
@@ -128,12 +129,14 @@ The MIT License (MIT). Please see [License File](LICENSE.md) for more informatio
 [ico-version]: https://img.shields.io/packagist/v/coconutcraig/laravel-postmark.svg?style=flat-square
 [ico-license]: https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square
 [ico-travis]: https://img.shields.io/travis/craigpaul/laravel-postmark/master.svg?style=flat-square
-[ico-scrutinizer]: https://img.shields.io/scrutinizer/coverage/g/coconutcraig/laravel-postmark.svg?style=flat-square
+[ico-style-ci]: https://styleci.io/repos/{id}/shield?branch=master[
```

Comment:
I think we have an out of place bracket at the end of this file. Also, here is the Style CI identifier since we need to make a change to this file anyway.

`80351847`

Reply:
Fixed